### PR TITLE
Opera.OperaCrypto version 94.0.4606.43

### PR DESCRIPTION
**Pull request by onyx-nxt**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/93459)

### CODE REVIEW

**Comment by KK-Designs** [comment]
@microsoft-github-policy-service agree

**Comment by KK-Designs** [comment]
Winget currently does not provide the info for Opera Crypto, so I am providing winget the info. I do not work at Opera i do not own the software, I am just adding it to the repo.

**Review by OverRevvv — DISMISSED** [review]

File: manifests/o/Opera/OperaCrypto/94.0.4606.43/Opera.OperaCrypto.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Opera.OperaCrypto
+PackageVersion: 94.0.4606.43
+PackageLocale: en-US
+Publisher: Opera Software
+PackageName: Opera Crypto Stable
+License: Proprietary software
```
Comment (stephengillie) [inline manifests/o/Opera/OperaCrypto/94.0.4606.43/Opera.OperaCrypto.locale.en-US.yaml:9]:
```suggestion
PublisherUrl: https://www.opera.com/
PublisherSupportUrl: https://www.opera.com/help
PrivacyUrl: https://legal.opera.com/privacy/
PackageName: Opera Crypto Stable
PackageUrl: https://www.opera.com/crypto/next
License: Proprietary software
LicenseUrl: https://legal.opera.com/eula/computers/
```

Comment (KK-Designs) [reply manifests/o/Opera/OperaCrypto/94.0.4606.43/Opera.OperaCrypto.locale.en-US.yaml]: I have made these changes and added some more

**Review by stephengillie — APPROVED** [review]

**Pull request merged**